src/entities/GroceryItem.ts

```diff
@@ -19,52 +18,38 @@ export class GroceryItem extends BaseEntity {
 
 	@Field(() => Number)
 	@Column({ type: 'int' })
-	groceryItemInfoId!: number; // foreign key to Fridge table
-
-	@Field(() => Number)
-	@Column({ type: 'int' })
-	groceryListId!: number; // foreign key to Fridge table
+	groceryListId!: number; // foreign key to GroceryList table
 
 	@Field(() => Number)
 	@Column({ type: 'int' })
 	quantity!: number;
 
-	// ---------------- relationship ----------------
-	@ManyToOne(() => GroceryList, groceryList => groceryList.groceryItems)
-	groceryList: GroceryList;
-
-	@ManyToOne(
-		() => GroceryItemInfo,
-		groceryItemInfo => groceryItemInfo.groceryItems
-	)
-	groceryItemInfo: GroceryItemInfo;
-
-	// ---------------- time ----------------
-	@Field(() => String)
-	@CreateDateColumn({ type: 'timestamptz' })
-	createdAt: Date;
-}
-
-@ObjectType()
-export class DetailedGroceryItem extends GroceryItem {
 	@Field(() => String)
+	@Column({ type: 'varchar' })
 	name!: string;
 
 	@Field(() => String, { nullable: true })
+	@Column({ type: 'varchar', nullable: true })
 	upc: string;
 
 	@Field(() => String, { nullable: true })
+	@Column({ type: 'varchar', nullable: true })
 	imgUrl: string;
 
 	@Field(() => Number)
+	@Column({ type: 'int' })
 	userId!: number; // foreign key to User table
 
 	@Field(() => Number)
+	@Column({ type: 'int' })
 	measurementTypeId!: number; // foreign key to MeasurementType table
 
-	@Field(() => String)
-	measurement!: string;
+	// ---------------- relationship ----------------
```

Comment:
added relationships for the foreign keys of measurementType & user on the next commit